plugin/evm/version.go

```diff
@@ -11,7 +11,7 @@ var (
 	// GitCommit is set by the build script
 	GitCommit string
 	// Version is the version of Subnet EVM
-	Version string
```

Comment:
I thought we typically provided this as a build flag? Why set it explicitly here?

Reply:
releaser does not set this correctly. we either had to tweak releaser, or set this explicitly. Coreth also sets this explicitly.

Reply:
Can't you just put it here? https://github.com/ava-labs/subnet-evm/blob/0abd6b6e55ba4eb0ef94cfa475b064eff64d6594/.goreleaser.yml#L6-L8

Reply:
Added